src/iocore/cache/StripeSM.cc

```diff
@@ -151,36 +162,11 @@ StripeSM::clear_dir()
 }
 
 int
-StripeSM::init(char *s, off_t blocks, off_t dir_skip, bool clear)
+StripeSM::init(bool clear)
 {
-  // Hash
-  char        *seed_str       = disk->hash_base_string ? disk->hash_base_string : s;
-  const size_t hash_seed_size = strlen(seed_str);
-  const size_t hash_text_size = hash_seed_size + 32;
-
-  hash_text = static_cast<char *>(ats_malloc(hash_text_size));
-  ink_strlcpy(hash_text, seed_str, hash_text_size);
-  snprintf(hash_text + hash_seed_size, (hash_text_size - hash_seed_size), " %" PRIu64 ":%" PRIu64 "",
-           static_cast<uint64_t>(dir_skip), static_cast<uint64_t>(blocks));
-  CryptoContext().hash_immediate(hash_id, hash_text, strlen(hash_text));
```

Comment:
Where this code goes?

Reply:
It moved to `Stripe::_init_hash_text`.

Reply:
It happened in 0412927.

Reply:
do you mean `Stripe::_init_hash_text` function? I can see `snprintf` makes the `hash_text`, but I can't find using it for `hash_id`. What I'm missing?
https://github.com/apache/trafficserver/pull/11811/files#diff-88d9e07d3781ce4d9c6307114be7856582eba25edc0d1c64ac9a73a59d00baecR102-R113